Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM node:7.8-alpine
```

Comment:
Sounds good. I'm not much of an expert on Docker, so can't really ring in on changes like that.